cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+  "baseUrl": "http://localhost:1234"
```

Comment:
LOL, can we serve from inside GH Actions? That's news for me...

Reply:
Actually this is for running it locally, we should be able to serve it but now that you brought this up I realized that the URL might be wrong, I'll try to run it locally to be sure